File: java-app/src/main/java/com/mercadolibre/java_app/review/ReviewContract.java
package com.mercadolibre.java_app.review;

import com.mercadolibre.java_app.commons.Purchase;
import com.mercadolibre.java_app.commons.ShippingOptionView;

public interface ReviewContract {
    interface ReviewView extends ShippingOptionView {
        void showProductPrice(String price);
        void showTotalPrice(String price);
        void goToCongrats(Purchase purchase);
    }
}

File: kt-app/src/main/java/com/mercadolibre/melikotlinmeetup/review/ReviewContract.kt
package com.mercadolibre.melikotlinmeetup.review

import com.mercadolibre.melikotlinmeetup.commons.Purchase
import com.mercadolibre.melikotlinmeetup.commons.ShippingOptionView

interface ReviewView: ShippingOptionView {
    fun showProductPrice(price: String)
    fun showTotalPrice(price: String)
    fun goToCongrats(purchase: Purchase)
}